doc/source/ray-air/getting-started.rst

```diff
@@ -173,7 +177,7 @@ Then use the ``Tuner`` to run the search:
 Batch Inference
 ~~~~~~~~~~~~~~~
 
-Use the trained model for scalable batch prediction with a ``BatchPredictor``.
+After running the steps in :ref:`Training <air-getting-started-training>` or :ref:`Tuning <air-getting-started-tuning>`, use the trained model for scalable batch prediction with a ``BatchPredictor``.
```

Comment:
nit: Should we rename `best_result` (from Tune) to `result` so this step would be compatible with running _just_ the Train step?

Reply:
updated